### ISSUE TITLE
Auto-Configuration does not work with Spring Boot 3

### ISSUE THREAD

**Issue by jochenberger** [JanLoebel/json-schema-validation-starter#9]
Auto-configuration must be done via `META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports` now.
> Rather than a single comma-separate list, each line contains the fully qualified name of an auto-configuration class.

https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.7-Release-Notes#changes-to-auto-configuration
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#auto-configuration-files

**Comment by jochenberger** [comment]
If I import the configuration manually, I still get errors in Spring Boot 3 though:
```
Caused by: java.lang.NoSuchMethodError: 'void org.springframework.web.server.ResponseStatusException.<init>(org.springframework.http.HttpStatus, java.lang.String)'
	at com.github.JanLoebel.jsonschemavalidation.JsonSchemaValidationException.<init>(JsonSchemaValidationException.java:15)
	at com.github.JanLoebel.jsonschemavalidation.provider.DefaultJsonSchemaProvider.handleValidationMessages(DefaultJsonSchemaProvider.java:31
...
```
I think that we might need to recompile against Spring 6.

**Comment by jochenberger** [comment]
Yeah, in spring-web 6, the `org.springframework.http.HttpStatus` implements an additional `org.springframework.http.HttpStatusCode` interface. I guess that trips the classloader.

**Comment by JanLoebel** [comment]
Thank you :) Hope it works, had not time to verify it further!

**Comment by jochenberger** [comment]
Oh, the build failed, I'm sorry.
https://jitpack.io/com/github/JanLoebel/json-schema-validation-starter/3.0.0/build.log
I guess Java 8 is not sufficient anymore.

**Comment by jochenberger** [comment]
#11 should fix that.

**Comment by jochenberger** [comment]
Looks better: https://jitpack.io/com/github/JanLoebel/json-schema-validation-starter/3.0.1/build.log

**Comment by jochenberger** [comment]
... and works just fine in my Spring Boot 3 project. Thanks! :+1:

**Comment by JanLoebel** [comment]
Great, and it was your effort, so thank you!
